PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/resources/META-INF/ruleConfigs.json

```diff
@@ -6,9 +6,14 @@
     ],
     "antipattern_message": "Azure Storage upload API without length parameter detected. Use upload API with length parameter instead."
   },
-
-  "ServiceBusReceiverAsyncClientCheck": {
-    "client_name": "ServiceBusReceiverAsyncClient",
-    "antipattern_message": "Use of ServiceBusReceiverAsyncClient detected. Use ServiceBusProcessorClient instead."
+  "DetectDiscouragedClientCheck": {
+    "ServiceBusReceiverAsyncClientCheck": {
+      "client_name": "ServiceBusReceiverAsyncClient",
+      "antipattern_message": "Use of ServiceBusReceiverAsyncClient detected. Use ServiceBusProcessorClient instead."
+    },
+    "EventHubConsumerAsyncClientCheck": {
+      "client_name": "EventHubConsumerAsyncClient",
+      "antipattern_message": "Use of EventHubConsumerAsyncClient detected. Use EventProcessorClient instead."
```

Comment:
We should make the message more informative, to include a few things on the lines
>provides a higher-level abstraction that simplifies event processing, making it the preferred choice for most developers.